### PR TITLE
Add pact tests for facility search

### PR DESCRIPTION
**Pull request by mpelzsherman**
## Description
Added pact tests for facility locator.

Also, removed a few vestiges of the v0 -> v1 cutover.

## Acceptance criteria
- [x] All facility locator API calls are covered by passing pact tests.

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: src/applications/facility-locator/tests/contract/search.pact.spec.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  },
+};
+
+contractTest('Facility Locator', 'VA.gov API', mockApi => {
+  describe('GET /v1/facilities/ccp', () => {
+    context('facilities: ccp data exists', () => {
+      it('responds with status 200 for pharmacy', async () => {
+        mockApi.addInteraction({
+          interaction,
+        });
```
Comment (U-DON) [inline src/applications/facility-locator/tests/contract/search.pact.spec.js:69]:
```suggestion
        mockApi.addInteraction(interaction);
```

File: src/applications/facility-locator/tests/contract/search.pact.spec.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+contractTest('Facility Locator', 'VA.gov API', mockApi => {
+  describe('GET /v1/facilities/ccp', () => {
+    before(() => {
+      window.disableMockLocatorApi = true;
+    });
+
+    context('facilities: ccp data exists', () => {
+      it('responds with status 200 for pharmacy', async () => {
+        mockApi.addInteraction(interaction);
```
Comment (U-DON) [inline src/applications/facility-locator/tests/contract/search.pact.spec.js:76]: Heads up that `mockApi` has changed to a [function that returns the actual mock API](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/contract-testing/how-to-use-pact-for-contract-testing.md#writing-a-contract-test):

```suggestion
        await mockApi().addInteraction(interaction);
```

**Comment by mpelzsherman** [comment]
@U-DON - When I run `yarn test:contract src/applications/facility-locator/tests/contract/search.pact.spec.js` now, it says "2 passing", even though I think I only have one test defined so far. What's up with that?

**Comment by U-DON** [comment]
If you've pulled in `master`, there is another contract test now. `yarn contract:test` runs all tests.

[To run a single test](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/contract-testing/how-to-use-pact-for-contract-testing.md#running-contract-tests), you want `yarn test:unit path/to/test.pact.spec.js`.

File: src/applications/facility-locator/api/index.js [hunk 3]
```diff
@@ -4,7 +4,13 @@ import environment from 'platform/utilities/environment';
...
 // To use vets-api data locally, replace the following with this:
 // export default LiveApi;
-export default ((environment.isLocalhost() || window.Cypress) &&
-!environment.API_URL.includes('review.vetsgov')
-  ? MockApi
-  : LiveApi);
+
+const getAPI = () => {
+  return !window.Mocha &&
+    (environment.isLocalhost() || window.Cypress) &&
```
Comment (brangi) [inline src/applications/facility-locator/api/index.js:10]: Would it be possible to convert this ternary into if statements? it's getting a bit difficult to read the logic

**Review by brangi — APPROVED** [review]

**Pull request merged**